composer.json

```diff
@@ -18,7 +18,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "react/promise": "^2.4"
```

Comment:
I'd prefer `suggest` here, as it's not a requirement, but then we can't have any version constraints.